### PR TITLE
Deploy Corda Enterprise

### PR DESCRIPTION
**Pull request by nadinequinr3**

### CODE REVIEW

File: content/en/platform/corda/5.0/deploying-operating/deployment/deploying/_index.md [hunk 1]
```diff
@@ -66,15 +70,60 @@ If not, then the following instructions describe how to push the images from the
...
+
+   docker tag postgres:14.4 $target_registry/postgres:14.4
+   docker push $target_registry/postgres:14.4
+   ```
+
+
 ## Download the Corda Helm Chart
 
-If you have access to Docker Hub, you can download the Corda Helm chart using the following command:
+If you have access to Docker Hub, you can download the Corda Helm chart using the following command for Corda Communnity:
```
Comment (yarivus) [inline content/en/platform/corda/5.0/deploying-operating/deployment/deploying/_index.md:115]:
```suggestion
If you have access to Docker Hub, you can download the Corda Helm chart using the following command for Corda Community:
```

**Review by yarivus — CHANGES_REQUESTED** [review]

**Review by yarivus — DISMISSED** [review]
LGTM

**Review by davidcurrie — COMMENTED** [review]
Looks good from my side.

@ronanbrowne - can you confirm that the names for the TAR files are correct and that customers will now pull the charts directly from Artifactory?

**Review by yarivus — APPROVED** [review]
LGTM

**Review by mcgovc — APPROVED** [review]
LGTM

**Review by amiegrace12 — APPROVED** [review]

**Pull request merged**